src/System.Management.Automation/engine/PSVersionInfo.cs

```diff
@@ -193,25 +193,6 @@ internal static string GitCommitId
             }
         }
 
-        internal static Version[] PSCompatibleVersions
-        {
-            get
-            {
-                return (Version[])s_psVersionTable[PSCompatibleVersionsName];
-            }
-        }
-
-        /// <summary>
-        /// Gets the edition of PowerShell.
-        /// </summary>
-        public static string PSEdition
```

Comment:
Is this added back somewhere further down? I can't really see anyone actually depending on this (e.g. why would this ever be `Desktop` externally, the class was only somewhat recently made public) it is technically a breaking change to remove it.

Reply:
You are right. I missed the fact that it's a public property. Will bring it back.